packages/cascara/src/components/Dashboard/Dashboard.module.scss

```diff
@@ -4,25 +4,17 @@ $widget-background: #f0f0f0;
 
 .Dashboard {
   display: block;
-  columns: 30rem;
+  // columns: 30rem; //verify we dont need this
   gap: $masonry-gutter;
 }
 
 .Widget {
-  margin-bottom: $masonry-gutter;
-  break-inside: avoid;
-
   &.scrolling .Data {
     overflow: auto;
   }
 
-  .Title {
-    display: block;
-    padding: 0.5em 0 0.25em;
-    margin: 0;
-  }
-
   .Data {
+    // we need this for Widget List
```

Comment:
If we remove this, the rows lines disappear, that's why I keep them:
![image](https://user-images.githubusercontent.com/87443054/151107565-9237a40b-1004-4b3e-8c17-151d32a4fe1d.png)

Reply:
This class is kept, no need to remove it.